pkg/DWSUtility.py

```diff
@@ -368,14 +368,14 @@ def do_resource_list(self):
 
             try:
                 crd_obj = self.dws.get_custom_resource_definition(crd)
-            except:
+            except Exception:
                 continue
             printer_cols = self.dws.get_crd_printer_columns(crd_obj)
```

Comment:
Should this be in the try? It seems like `crd_obj` could be `None` without an exception thrown: https://github.com/kubernetes-client/python/blob/e3c0042e0fa69d72d5e4a9dfd481a0f896a67b66/kubernetes/client/api_client.py#L194.

Reply:
`get_custom_resource_definition` will throw if `crd_name` doesn't exist.